### PR TITLE
Big number separating

### PR DESCRIPTION
**Pull request by AscuasFunkeln**
Using commas separating for big number in skills window to make the numbers more readable.
Updated latest file version.

### CODE REVIEW

**Comment by diath** [comment]
This function produces trailing `.0` for natural numbers, it probably shouldn't or it will look silly.

**Comment by AscuasFunkeln** [comment]
I just make it for skills in my project and work fine, it dont return any `.0`
Yeah, better idea is moving it to make this useable in every modules.

File: modules/gamelib/util.lua [hunk 1]
```diff
@@ -8,4 +8,8 @@ function dirtostring(dir)
       return k
     end
   end
-end
\ No newline at end of file
+end
```
Comment (diath) [inline modules/gamelib/util.lua:11]: Newline after this end?

**Comment by diath** [comment]
> I just make it for skills in my project and work fine, it dont return any .0

This is true in Lua 5.1 and Lua 5.2 but not Lua 5.3.

**Comment by AscuasFunkeln** [comment]
> > I just make it for skills in my project and work fine, it dont return any .0
> 
> This is true in Lua 5.1 and Lua 5.2 but not Lua 5.3.

`return math.ceil(left..(num:reverse():gsub('(%d%d%d)','%1,'):reverse())..right)`
This will dont solve this trailing?

**Comment by diath** [comment]
No, but I guess we can merge it as-is since it works fine with LuaJIT (our preferred Lua implementation) and Lua 5.1 which is the fallback in the CMake project when LuaJIT is not found.

**Comment by scopz** [comment]
Isn't this behavior controller by the "tr" function?
I'm asking because locales should be taken into account when formatting numbers.
In Spanish (not sure about Portuguese) thousandSeparators are dots (.) and decimalSeparators (,) and this "tr" function checks this.
image

**Comment by diath** [comment]
It looks like that was intended with the `tr` function but I don't think it's actually used for number values that much in modules, I guess all of them would have to be revised. Additionally, you would have to figure out what to do with locale strings that contain numbers on their own.

**Pull request merged**